storage/innobase/srv/srv0srv.cc

```diff
@@ -1479,6 +1485,29 @@ srv_export_innodb_status(void)
 
 	export_vars.innodb_buffer_pool_pages_dirty = flush_list_len;
 
+	export_vars.innodb_buffer_pool_pages_made_young = stat.n_pages_made_young;
+	export_vars.innodb_buffer_pool_pages_made_not_young = stat.n_pages_not_made_young;
+	export_vars.innodb_buffer_pool_pages_old = 0;
+	for (i = 0; i < srv_buf_pool_instances; i++) {
+		buf_pool_t*	buf_pool = buf_pool_from_array(i);
+		export_vars.innodb_buffer_pool_pages_old
+			+= buf_pool->LRU_old_len;
+	}
```

Comment:
We are missing the equivalent of
```c++
	export_vars.innodb_buffer_pool_pages_LRU_flushed =
		stat.buf_lru_flush_page_count;
```
I think that we should just tie the pointer to the variable that is updated by `buf_flush_LRU_list_batch()`.

Reply:
It is admittedly a little questionable to export a pointer to a shared variable and then let it be read without appropriate concurrency protection, but given that `sizeof(ulint)` is the CPU register width, we should not have a problem with torn reads. Also, we do not care if the value is a little stale.